var/spack/repos/spack_repo/builtin/packages/py_vesin/package.py

```diff
@@ -0,0 +1,41 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.python import PythonPackage
+
+from spack.package import *
+
+
+class PyVesin(PythonPackage):
+    """Computing neighbor lists for atomistic system."""
+
+    homepage = "https://luthaf.fr/vesin/latest/index.html"
+    pypi = "vesin/vesin-0.3.7.tar.gz"
+
+    import_modules = ["vesin"]
+
+    maintainers("HaoZeke", "luthaf", "rmeli")
+    license("BSD-3-Clause", checked_by="HaoZeke")
+
+    version("0.3.7", sha256="52c11ac0ba775c228f06779877cf8641854edab7ea59036093ef5e8447379de0")
+    extends("python@3.9:")
+
+    variant("ase", default=False, description="With ASE")
+    variant("metatensor", default=False, description="With metatensor learn, torch")
+    variant("metatomic", default=False, description="With metatomic")
+    variant("torch", default=False, description="With PyTorch")
```

Comment:
None of these variants are listed in pyproject.toml, I wouldn't add them here either so that we are consistent with pip